kde/ki18n/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+# Based on Portfile for kde-cmake-extra-modules
+
+# Potentially to be replaced as part of
+# https://trac.macports.org/ticket/49925
+
+PortSystem          1.0
+PortGroup           qt5     1.0
+PortGroup           cmake   1.1
+
+name                ki18n
+subport ${name}-devel {}
+
+set kf5.branch      5.47
+
+if {${subport} eq "${name}-devel"} {
```

Comment:
There is no point in creating a -devel port for KI18n. It's fine (for a -hopefully- temporary KI18n port) to adapt the ECM Portfile, but I suggest you clean it up.